sector-builder-ffi/examples/simple/porcelain.rs

```diff
@@ -88,10 +100,42 @@ pub(crate) unsafe fn resume_seal_sector_nonblocking(
     thread::spawn(move || {
         let sector_builder = atomic_ptr.into_inner();
 
-        let _ = resume_seal_sector(&mut Default::default(), sector_builder, sector_id);
+        let _ = resume_seal_pre_commit(&mut Default::default(), sector_builder, sector_id);
     });
 }
 
+pub(crate) unsafe fn seal_commit_nonblocking(
+    ptr: *mut sector_builder_ffi_SectorBuilder,
+    sector_id: u64,
+    seed: sector_builder_ffi_FFISealSeed,
+) {
+    let atomic_ptr = AtomicPtr::new(ptr);
+
+    thread::spawn(move || {
+        let sector_builder = atomic_ptr.into_inner();
+
+        let _ = seal_commit(&mut Default::default(), sector_builder, sector_id, seed);
```

Comment:
don't you want to log the error at least somewhere? or is that caught somewhere else?

Reply:
fixed